Classes/BEMCheckBox.m

```diff
@@ -172,6 +172,27 @@ - (void)handleTapCheckBox:(UITapGestureRecognizer *)recognizer {
     }
 }
 
+// increase touch area
+- (BOOL) pointInside:(CGPoint)point withEvent:(UIEvent *)event;
+{
+    BOOL found = [super pointInside:point withEvent:event];
+    
+    CGFloat minimumSize = 44;
+    CGFloat w = self.frame.size.width;
```

Comment:
To conform with the rest of the variable names in the project, please use the full word "width".